index.bs

```diff
@@ -257,17 +260,28 @@ The <dfn export>previous frame starting point</dfn> of a {{Node}} |N| in a
 coordinate space |C| is the point which, <a>in the previous frame</a>, was the
 <a>starting point</a> of |N| in |C|.
 
+The <dfn export>previous frame transform-indifferent starting point</dfn> of a
+{{Node}} |N| in a coordinate space |C| is the point which, <a>in the previous
+frame</a>, was the <a>transform-indifferent starting point</a> of |N| in |C|.
+
 The <dfn export>previous frame visual representation</dfn> of a {{Node}} |N| is
 the set which, <a>in the previous frame</a>, was the <a>visual
 representation</a> of |N|.
 
+Point |A| <dfn>differs significantly</dfn> from point |B| if |A| and |B|
+differ by 3 or more <a>pixel units</a> in either the horizontal or vertical
+direction.
+
 Unstable Nodes {#sec-unstable-nodes}
 ------------------------------------
 
-A {{Node}} |N| <dfn export>has shifted</dfn> in a coordinate space |C| if the
-<a>starting point</a> of |N| in |C| differs from the <a>previous frame starting
-point</a> of |N| in |C| by 3 or more <a>pixel units</a> in either the horizontal
-or vertical direction.
+A {{Node}} |N| <dfn export>has shifted</dfn> in a coordinate space |C| if:
+
+* the <a>starting point</a> of |N| in |C| <a>differs significantly</a> from the
```

Comment:
I think we should consider transform-indifferent starting point for "move vector", too. The chromium CL does that, and it's not easy for it to consider transform-indifferent starting point when judging shift, but not when calculating move vector.